Library/Formula/choose-gui.rb

```diff
@@ -0,0 +1,18 @@
+class ChooseGui < Formula
+  desc "Hybrid string fuzzy matcher that uses both std{in,out} and a native GUI"
+  homepage "https://github.com/sdegutis/choose"
+  url "https://github.com/sdegutis/choose/archive/1.0.tar.gz"
+  sha256 "b1d16c6e143e2a9e9b306cd169ce54535689321d8f016308ff26c82c3d2931bf"
+
+  conflicts_with "choose", :because => "both install a `choose` binary"
```

Comment:
Mind flipping these around with a space?

```ruby
depends_on :xcode => :build

conflicts_with "choose", :because => "both install a `choose` binary"
```
Thanks!